src/content/en/2020/css.md

```diff
@@ -1268,27 +1338,42 @@ There is so much discussion (or argument) about CSS-in-JS that one could assume
+
+To support bidirectionality, explicit support for indicating direction is available in both HTML via ([the `dir` attribute](https://html.spec.whatwg.org/multipage/dom.html#the-dir-attribute) and the [`<bdo>` element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-bdo-element)), and CSS (the [direction](https://www.w3.org/TR/css-writing-modes-3/#direction) and [`unicode-bidi`](https://www.w3.org/TR/css-writing-modes-3/#unicode-bidi) properties. We looked at usage of both HTML and CSS methods.
 
-To support bidirectionality,  explicit support for indicating direction is available in both HTML via ([the `dir` attribute](https://html.spec.whatwg.org/multipage/dom.html#the-dir-attribute) and the [`<bdo>` element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-bdo-element)), and CSS (the [direction](https://www.w3.org/TR/css-writing-modes-3/#direction) and [`unicode-bidi`](https://www.w3.org/TR/css-writing-modes-3/#unicode-bidi) properties. We looked at usage of both HTML and CSS methods.
-Only 12.14% of pages on mobile (and a similar 10.76% on desktop) set the `dir` attribute on the `<html>` element. Which is fine: most writing systems in the world are **ltr**, and the default `dir` value is **ltr**. Of the pages which *did* set `dir` on `<html>`, 91% set it to `ltr` while 8.5% set it to `rtl` and 0.32% to `auto` (the explicit direction is unknown value, mainly useful for templates which will be filled with unknown content). An even smaller number, 2.63%, set `dir` on `<body>`. Which is good, because setting it on `<html>` also covers you for content in the `<head>`, like `<title>`.
+Only 12.14% of pages on mobile (and a similar 10.76% on desktop) set the `dir` attribute on the `<html>` element. Which is fine: most writing systems in the world are `ltr`, and the default `dir` value is `ltr`. Of the pages which did set `dir` on `<html>`, 91% set it to `ltr` while 8.5% set it to `rtl` and 0.32% to `auto` (the explicit direction is unknown value, mainly useful for templates which will be filled with unknown content). An even smaller number, 2.63%, set `dir` on `<body>`. Which is good, because setting it on `<html>` also covers you for content in the `<head>`, like `<title>`.
 
-Why set direction using HTML attributes rather than CSS styling? One reason is separation of concerns: direction has to do with content which is the purview of HTML. It is also the [recommended practice](https://www.w3.org/International/tutorials/bidi-xhtml/index.en): *“Avoid using CSS or Unicode control codes for managing direction where you can use markup”*. After all, the stylesheet might not load and the text still needs to be readable.
+{# TODO Authors: I can't find this 2.63% number in the data (https://docs.google.com/spreadsheets/d/1sMWXWjMujqfAREYxNbG_t1fOJKYCA6ASLwtz4pBQVTw/edit#gid=368716708) and it's not smaller than 0.32% so bit confused??? #}
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
Um, I don't immediately find it either. Will check more when I have time, I need to start packing for  travel soon, sorry!

Reply:
@svgeesus did you get a chance to look at this?

Reply:
I found it @svgeesus . Had to pivot the table to add up all the values and then devide by the total number of pages crawled. Add the table [in the sheet](https://docs.google.com/spreadsheets/d/1sMWXWjMujqfAREYxNbG_t1fOJKYCA6ASLwtz4pBQVTw/edit#gid=368716708) in case anyone else goes looking at changed:

> An even smaller number, 2.63%, set `dir` on `<body>`

to:

> An even smaller number, 2.63%, set `dir` on `<body>` than on `<html>`

to make it more obvious were talking about the whole HTML numbers and not just the 0.32 that use auto.

Resolving.